### ISSUE TITLE
IOException ("More data is available") When building within a Windows Container on Docker Desktop

### ISSUE THREAD

**Issue by vindiagram** [fabric8io/docker-maven-plugin#1441]
### Description
I have a project that builds a windows docker container using the docker-maven-plugin. This image is intended to be used as a build agent for Jenkins. This image builds fine on the windows host with no errors.
But when I attempt to build this same project within a container based on the already built image the docker-maven-plugin throws an IOException ("More data is available") during BuildService.buildImage on queryService.getImageId.

When running this container I volume mount  "-v \\.\pipe\docker_engine:\\.\pipe\docker_engine"

I can't share the project because its is on our companies internal network.

I can share that our base image is: **mcr.microsoft.com/dotnet/framework/runtime:3.5-20210209-windowsservercore-ltsc2019**
We are using maven to pull from our artifactory "Install Shield 2020" and "Visual Studio 2019 (offline install created from /layout feature)". There are other files but these are the largest at about 3.5 GB. These are added to the container via an assembly script.

But at this point when building from within the container, in a shared volume or not the image fails on inspect with the above error before attempting to build.

Also I am able to install docker-cli with chocolatey and run docker build in the target directory built by the plugin and it is able to complete. So I can say this is an issue within the "docker-maven-plugin" and not an issue communicating with the docker host through the \\.\pipe\docker_engine.

### Info
* Docker Desktop: 3.2.2
  * Engine 20.10.5
* Maven: 3.6.3
* JDK: 1.8.0
* docker-maven-plugin version : 0.34.1

### Stack Trace:
```
Caused by: io.fabric8.maven.docker.access.DockerAccessException: Unable to inspect image [<our image name>]
[More data is available] : More data is available
    at io.fabric8.maven.docker.access.hc.DockerAccessWithHcClient.inspectImage (DockerAccessWithHcClient.java:446)
    at io.fabric8.maven.docker.access.hc.DockerAccessWithHcClient.getImageId (DockerAccessWithHcClient.java:433)
    at io.fabric8.maven.docker.service.QueryService.getImageId (QueryService.java:135)
    at io.fabric8.maven.docker.service.BuildService.buildImage (BuildService.java:151)
    at io.fabric8.maven.docker.service.BuildService.buildImage (BuildService.java:71)
    at io.fabric8.maven.docker.BuildMojo.proceedWithDockerBuild (BuildMojo.java:104)
    at io.fabric8.maven.docker.BuildMojo.proceedWithBuildProcess (BuildMojo.java:92)
    at io.fabric8.maven.docker.BuildMojo.buildAndTag (BuildMojo.java:85)
    at io.fabric8.maven.docker.BuildMojo.processImageConfig (BuildMojo.java:157)
    at io.fabric8.maven.docker.BuildMojo.executeInternal (BuildMojo.java:74)
    at io.fabric8.maven.docker.AbstractDockerMojo.execute (AbstractDockerMojo.java:251)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
```

**Comment by rohanKanojia** [comment]
Hmm, I don't have access to a Windows machine so I'm not sure I will be able to reproduce this issue. Would it be possible for you to debug docker-maven-plugin to see what's actually causing the issue?

**Comment by vindiagram** [comment]
Yes i have been trying to debug through mvnDebug, I did narrow it down to "EntityUtils.toString" is throwing the More data is available IOException called by ApacheHttpClientDelegate.getResponseMessage. But its not very helpful, I am continuing to look into the issue I was just hoping if I opened the ticket I could get some backup. I do see the the message is "chunked" so that could be related to the issue, it sounds as if it has not entirely consumed the message from the stream by the time the plugin tries to consume the response. But i though the httpclient library should handle this, perhaps ill try updating to the latest version

**Comment by rohanKanojia** [comment]
Thanks a lot :pray: .
